package.json

```diff
@@ -24,9 +24,8 @@
     "e2e-prepare": "selenium-standalone install",
     "e2e-server": "node ./bin/server.js",
     "test": "standard && node ./bin/test.js",
-    "test-full": "zuul --concurrency 1 -- test/*.spec.js",
-    "test-local": "zuul --local 11111 --no-coverage -- test/*.spec.js",
-    "test-phantom": "zuul --phantom -- test/*.spec.js"
+    "test-full": "karma start --browserStack.startTunnel 0 --browserStack.tunnelIdentifier $BROWSERSTACK_LOCAL_IDENTIFIER",
```

Comment:
I'm not entirely sure why `$BROWSERSTACK_LOCAL_IDENTIFIER` is needed. While setting up browserstacklocal I ended up having two env variables with the secret key in. This being one of them, and `BROWSER_STACK_ACCESS_KEY` (which `browserstacklocal` or `karma-browserstack` complained about not existing).

Reply:
I _think_ it's because `$BROWSERSTACK_LOCAL_IDENTIFIER` is for Travis, but I'm not sure.